FlaskApp/app/static/js/script_barchart.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by iris on 2/20/16.
+ */
+$(function () {
+    $('#container').highcharts({
```

Comment:
I feel you can do a little more functional? pass div id to the method and then call it. 
same reason.. hardcode.
btw the div id  is container? I think you used it somewhere else already.. div id should be unique.